netebpfext/user/fwp_um.h

```diff
@@ -37,27 +37,122 @@ typedef class _fwp_engine
         return id;
     }
 
+    FWPS_CALLOUT3*
+    get_fwps_callout_under_lock(const GUID* callout_key)
```

Comment:
Add SAL annotation?  Or does this need to literally match what is in a windows header file? (If the latter, then ok as is)